jboss-fuse-sap-tool-suite/site/pom.xml

```diff
@@ -12,5 +12,8 @@
 	<artifactId>org.fusesource.ide.sap.updatesite</artifactId>
 	<packaging>eclipse-repository</packaging>
 	<name>JBoss Tools Fuse Extras :: SAP Tooling :: Update Site</name>
+	<properties>
+		<skipDeployToJBossOrg>false</skipDeployToJBossOrg>

```

Comment:
why do we need to add this new property?
I don't understand the link to [JBIDE-24788]